test/topology_experimental_raft/suite.yaml

```diff
@@ -9,3 +9,5 @@ extra_scylla_config_options:
 skip_in_release:
   - test_blocked_bootstrap
   - test_raft_cluster_features
+skip_in_debug:
+  - test_raft_cluster_features
```

Comment:
Actually, there is one more cluster features pytest: `test/topology_custom/test_deprecating_cluster_features.py`. It only has a single test case which should run quickly, but for consistency you should disable it, too.